src/components/Filters.jsx

```diff
@@ -0,0 +1,143 @@
+                            <span className="genre-text">
+                                4 stars &amp; above
+                            </span>
+                        </label>
+                    </li>
+                    <li className="list-item mb-0-5">
+                        <label htmlFor="rating-3-above" className="text-sm rating-label flex-row flex-align-center">
+                            <input type="radio" className="input-radio" id="rating-3-above" name="product-ratings" />
+                            <span className="genre-text">
+                                 3 stars &amp; above
+                            </span>
+                        </label>
+                    </li>
+                    <li className="list-item mb-0-5">
+                        <label htmlFor="rating-2-above" className="text-sm rating-label flex-row flex-align-center">
+                            <input type="radio" className="input-radio"  id="rating-2-above" name="product-ratings"/>
+                            <span className="genre-text">
+                                Ratings 2 &amp; above
+                            </span>
+                        </label>
+                    </li>
+                    <li className="list-item mb-0-25">
+                        <label htmlFor="rating-1-above" className="text-sm rating-1-above rating-label flex-row flex-align-center">
+                            <input type="radio" className="input-radio" id="rating-1-above" name="product-ratings" />
+                            <span className="genre-text">
+                                Ratings 1 &amp; above
+                            </span>
+                        </label>
+                    </li>
+                </ul>
+            </div>
+            <div className="filter-category mt-1 mb-2">
+                <ul className="list-spaced list-style-none">
+                    <h6 className="filter-head text-reg list-head px-0-5 py-0-25 mb-0-5">
+                        Book Cover
+                    </h6>
+                    <li className="list-item mt-0-75 mb-0-5">
+                        <label htmlFor="cover-paperback" className="text-sm book-cover-label flex-row flex-align-center">
+                            <input type="checkbox" className="input-checkbox" id="cover-paperback" />
+                            <span className="genre-text">
+                                Paperback
+                            </span>
+                        </label>
+                    </li>
+                    <li className="list-item my-0-5">
+                        <label htmlFor="cover-hardcover" className="text-sm book-cover-label flex-row flex-align-center">
+                            <input type="checkbox" className="input-checkbox" id="cover-hardcover" />
+                            <span className="genre-text">
+                                Hardcover
+                            </span>
+                        </label>
```

Comment:
Similarly for categories too. You may want to add new categories in future, then you've to manually add all the lines of code, the other way round you're already getting categories from api just loop over it. Just a suggestion.

Reply:
This as well. I shall make this change once I'm done with the basic P0 requirements!

Thank you for your review!